writers/writer.go

```diff
@@ -76,3 +90,70 @@ func (c *consumer) channelExists(channel string) bool {
 	_, found := c.channels[channel]
 	return found
 }
+
+func (c *consumer) subtopicExists(subtopic string) bool {
+	if _, ok := c.subtopics["*"]; ok {
+		return true
+	}
+
+	_, found := c.subtopics[subtopic]
+	return found
+}
```

Comment:
```
for _, s := range subtopics {
    if s.Key == subtopic ||  s.Key == "*" {
        return true
    }
    return false
}
```

Reply:
I can't use `s.Key`. Have you idea how to fix that?

Reply:
@manuio I can't use `s.Key`. Have you idea how to fix that?

Reply:
Edit: I just need to remove `.Key` and it is OK. But there is another interrogation:
`for _, s := range c.subtopics {
	if s == subtopic ||  s == "*" {
		return true
	}
	return false
}
// Missing return call
`
Do I need to `return false` at the place of comment?

Reply:
`for _, s := range c.subtopics {
	if s == subtopic ||  s == "*" {
		return true
	}
	return false
}
// Missing return call`

Reply:
@jonathandreyer No, you are right. I copy pasted a piece of code but it was not complete. I just edited my previous comment, now it should be ok.